manticore/platforms/linux.py

```diff
@@ -387,6 +392,7 @@ class Linux(Platform):
 
     # from /usr/include/asm-generic/resource.h
     RLIMIT_NOFILE = 7  # /* max number of open files */
+    RLIMIT_STACK = 3
```

Comment:
Loud thinking: we could probably use https://docs.python.org/3/library/resource.html#resource-limits ?